### DISCUSSION TITLE
Link option in card to "force" open in new tab

### DISCUSSION THREAD

**Discussion by StefanKoell** [imfing/hextra#237] [Ideas]
Hi!

When using cards for local/bundled images in blog posts or documentation, it would be nice to have an option to force open those linked images in a new tab (target="_blank").

Or is this already possible and I'm missing something?

Regards,
Stefan

**Comment by imfing** [comment]
@StefanKoell Hi Stefan, by default clicking the card opens the `link` property in a new tab. 

```
{{< cards >}}
  {{< card link="/" title="Local Image" image="images/space.jpg" subtitle="Image under assets directory, processed by Hugo." method="Resize" options="600x q80 webp" >}}
{{< /cards >}}
```


> linked images in a new tab

Can you provide an example for opening a linked images? Did you mean opening the `image` source file if `link` is not provided?
